datasets/persian_ner/persian_ner.py

```diff
@@ -0,0 +1,169 @@
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+The dataset includes 250,015 tokens and 7,682 Persian sentences in total. It is available in 3 folds to be used in turn as training and test sets. The NER tags are in IOB format.
+"""
+
+_HOMEPAGE = ""
+
+_LICENSE = "Creative Commons Attribution 4.0 International License"
+
+_URL = "https://github.com/HaniehP/PersianNER/raw/master/ArmanPersoNERCorpus.zip"
+
+
+class PersianNER(datasets.GeneratorBasedBuilder):
+    """ArmanPerosNERCorpus - the first manually-annotated Persian NER corpus."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="fold1", version=VERSION, description="This is the first fold  of the ArmanPersoNERCorpus"
+        ),
+        datasets.BuilderConfig(
+            name="fold2", version=VERSION, description="This is the second fold  of the ArmanPersoNERCorpus"
+        ),
+        datasets.BuilderConfig(
+            name="fold3", version=VERSION, description="This is the third fold  of the ArmanPersoNERCorpus"
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "fold1"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "tokens": datasets.Sequence(datasets.Value("string")),
+                "ner_labels": datasets.Sequence(
```

Comment:
```suggestion
                "ner_tags": datasets.Sequence(
```

Can you rename this fields `ner_tags` to match the other NER datasets format ?